src/components/AffectedOfferingForm.vue

```diff
@@ -31,12 +31,29 @@ const isScreenSortaSmall = computed(() => screenWidth.value < 950);
 
 const modelValue = defineModel<ZodAffectType>();
 
-const affectCvss3Vector = computed(
-  () => modelValue.value?.cvss_scores.find(({ issuer, cvss_version }) => issuer === 'RH' && cvss_version === 'V3')
-    ?.vector
-    || null
+const defaultRhCvss3 = {
+  cvss_version: 'V3',
+  issuer: 'RH',
+  comment: '',
+  score: null,
+  vector: '',
+  embargoed: modelValue.value?.embargoed,
+  alerts: [],
+};
```

Comment:
This same object is used in `useFlawAffectsModel` and a couple of different tests files, so maybe we could export this somewhere?